pull_request_template.md

```diff
@@ -0,0 +1,20 @@
+<!-- Add a meaningful description of the changes you made -->
+## Summary
+This PR is a part of the work to resolve #_issue
+....
+
+<!-- Some description of HOW you achieved it. Perhaps give a high-level description of the program flow. Did you need to refactor something? What tradeoffs did you take? Are there things in here that you’d particularly like people to pay close attention to? -->
+## Changes
+-
+-
+
+<!-- Go through the checklist below to verify that your PR is good and ready for review -->
+
+## Checks to complete
+- [ ] I performed a self-review of my code.
+- [ ] PR title is in a conventional commit style.
+- [ ] My code follows the style guide of the project, I've built and run it, and no warnings or errors occurred.
+- [ ] I documented my code properly.
+- [ ] I added tests for my code.
+- [ ] Existing tests passed with my changes.
+- [ ] I set up PR settings properly (assignee, reviewers, link project, set milestone).
```

Comment:
nit:

```suggestion
- [ ] I've built and run my changes, and no warnings or errors occurred.
- [ ] All existing tests passed with my changes.
- [ ] My code follows our style guide.
- [ ] I've performed a self-review of my code.
- [ ] PR's title has a conventional commit style.
- [ ] Every non-private interface is documented properly.
- [ ] I've added tests for my code (if it makes sense).
- [ ] PR has assignee, reviewers, milestone; is linked to the project, and issue).
```

Reply:
Thanks for your advice, fixed with https://github.com/ios-course/ironfoudation-team-project/pull/53/commits/2c762c3cf6ae98cc13ed4b998f3e3cff9c42163c